app/validators/has_one_title_validator.rb

```diff
@@ -0,0 +1,8 @@
+# validates that the title has at least one title
+class HasOneTitleValidator < ActiveModel::Validator
```

Comment:
Yeah. @cbeer Can you get rid of https://github.com/projecthydra-labs/curation_concerns/blob/curation_concerns_models/app/has_one_title_validator.rb ?